README.md

```diff
@@ -31,6 +31,17 @@ You can see the full list of mintlayer executables by running `cargo run --bin`,
 - `wallet-rpc-daemon`: A wallet that can be controlled remotely using RPC. This can be run as a service.
 - `wallet-cli`: A command line interface for the wallet. All the newest features and functionalities of the wallet are available in this wallet. The `wallet-cli` is extremely ubiquitous and can be run in many modes, including a self-contained wallet, a controller of an RPC wallet, as an RPC wallet itself, and also as a cold wallet for air-gapped storage of coins and assets.
 
+### What do you need to run?
+
+There are many ways you can manage running the components of mintlayer-core. The following are a few examples.
+
+You can:
+
+- Run the node-daemon + wallet-cli to manage your coins, where the wallet-cli owns your wallet file and manages your coins. The wallet-cli itself can optionally act as an RPC server
+- Run the node-daemon + wallet-rpc-daemon + wallet-cli, where both daemons will be services in your system, the wallet-rpc-daemon will own your wallet (and start staking automatically for you), and the wallet-cli will just be an easy interface for communicating with your wallet
+- Run only the node-gui, where the GUI contains both the node and the wallet. This is a good option for non-techy users
+- Run the node-daemon with the API server components ([see this readme](api-server/README.md)) without a wallet, to manage your application
```

Comment:
A nitpick: I wouldn't use `the` with the names of applications, because they are basically proper names (I mean technically a proper name can be used with `the` in English, like `The Eiffel Tower`, but an application name is more like a person's name to me and `the` is not used with those). Alternatively, I'd remove the hyphen to make the name a common name. So, I'd write

> Run the node daemon + wallet cli to manage your coins, where the wallet cli owns your wallet file and manages your coins.

or

> Run node-daemon + wallet-cli to manage your coins, where wallet-cli owns your wallet file and manages your coins.

Reply:
Also, there are no full stops at the end of the sentences.

Reply:
Fixed. The full stops though is intended because it's a list. AFAIK, lists don't follow these rules. Not sure to be honest.